cli/src/pcluster/models/imagebuilder.py

```diff
@@ -150,4 +170,23 @@ def __init__(
         self.image = image
         self.build = build
         self.dev_settings = dev_settings
-        self._add_validator(BaseAMIValidator, ami_id=self.build.parent_image)
+        self._set_default()
+        self._add_validator(
+            EbsVolumeTypeSizeValidator, priority=10, volume_type=Param("gp2"), volume_size=self.image.root_volume.size
+        )
+
+    def _set_default(self):
+        # set default root volume
+        if self.image.root_volume is None or self.image.root_volume.size.value is None:
+            increase_volume_size = 15
+            ami_id = utils.get_ami_id_from_parent_image(self.build.parent_image.value)
+            ami_info = utils.get_info_for_amis([ami_id])
+            default_root_volume_size = (
+                ami_info[0].get("BlockDeviceMappings")[0].get("Ebs").get("VolumeSize") + increase_volume_size
+            )
+            if self.image.root_volume is None:
+                default_root_volume = Volume(size=default_root_volume_size)
+                default_root_volume.implied = True
```

Comment:
We don't need to use the implied for objects, it's only useful for the params, so the size in this case. You should do: `volume.size.implied = True` instead. 
If we're able to move this intiialization in the Volume as mentioned before the set of the implied would be automatic.